### PR TITLE
Use checkbox for measure buttons

### PR DESCRIPTION
**Pull request by ger-benjamin**
Fix: https://github.com/camptocamp/ngeo/issues/684

Example: https://ger-benjamin.github.io/ngeo/measure_options_layout/examples/contribs/gmf/apps/mobile/index.html?lang=en

### CODE REVIEW

**Comment by pgiraud** [comment]
The result looks perfect to me.
Wouldn't it better to change the HTML to avoid using .btn and .btn-default at all?

**Comment by ger-benjamin** [comment]
Yes, but that seems generated by the ngeo-btn directive...

**Comment by ger-benjamin** [comment]
Done

**Comment by pgiraud** [comment]
I would do more job in the HTML.
https://github.com/camptocamp/ngeo/compare/master...pgiraud:measure_options_layout
What do you think?

**Comment by sbrunner** [comment]
Less we have in the HTML better it is, we shouldn't forget that the HTML is copy on all the client application, than he will be harder to maintain ...

**Comment by sbrunner** [comment]
Be had a call with @pgiraud and finally it's not so hard HTML changes, it's OK for me :-)

File: contribs/gmf/apps/mobile/index.html [hunk 1]
```diff
@@ -108,11 +108,11 @@
       <div id="measure-tools" class="slide" data-header-title="Measure tools">
         <a ngeo-btn
           ng-click="mainCtrl.hideNav()"
-          class="btn btn-default measure-tools-button"
+          class="measure-tools-button"
           ng-model="mainCtrl.measurePointActive">Coordinate</a>
```
Comment (sbrunner) [inline contribs/gmf/apps/mobile/index.html]: Missing translate

**Pull request closed without merging**